packages/@cdktf/provider-generator/lib/get/generator/emitter/resource-emitter.ts

```diff
@@ -123,6 +123,8 @@ export class ResourceEmitter {
     this.code.line(`dependsOn: config.dependsOn,`);
     this.code.line(`count: config.count,`);
     this.code.line(`lifecycle: config.lifecycle,`);
+    this.code.line(`provisioners: config.provisioners,`);
+    this.code.line(`connection: config.connection,`);
```

Comment:
With this change, we need to make sure that we only release this code with 0.12 and not as a patch before that (but that is already true for the work on iterator that has been merged, I think).